sunpy/physics/differential_rotation.py

```diff
@@ -481,7 +481,7 @@ def _warp_sun_coordinates(xy, smap, new_observer, **diff_rot_kwargs):
             coordinates_at_map_observer = rotated_coord.transform_to(smap.coordinate_frame)
 
         # Go back to pixel coordinates
-        x2, y2 = smap.world_to_pixel(coordinates_at_map_observer)
+        x2, y2 = smap.wcs.world_to_pixel(coordinates_at_map_observer)
```

Comment:
Since this has changed, the places where x2 and y2 are used, need to be updated.

Reply:
Something has to be changed to make the code work again. If x2 and y2 here arent, then what should we change?

Reply:
world_to_pixel always returns two values. That can either be assigned to one variable or two, like in this case.

We don't need to worry about pixel_to_world

Reply:
The changelog has not been reverted.

The fix can not be using pixelpair, we have to change other parts of the code.